### ISSUE TITLE
Sorting variables according to Data Category

### ISSUE THREAD

**Issue by niva83** [wind-energy/variables-dictionary#1]
@pgancarski ,

To link variables-dictionaries to the taxonomies we could aim at grouping variables according to the taxonomy of data categories, which is:

https://github.com/wind-energy/taxonomies-and-vocabularies/blob/master/taxonomy-data-category-simple.json

In that case we can point from / to taxonomy to/from variables dictionary.

**Comment by pgancarski** [comment]
Yes, what we could do is add a data_category field to each variable. Then based on that the data converter could automatically tag a data-set.

TODO:
- fix the schema - https://github.com/wind-energy/variables-dictionary/blob/master/schema.json
- update the dictionaries
- update the converter to support auto tagging
